OHM-Trade-Agent-v1/tests/test_opip_confidence_evidence_v1.py

```
@@ -363,3 +370,115 @@ def test_recent_funnel_skips_malformed_paper_admission_values(tmp_path):
+    assert report["early_watch_observed"] == 2
+    assert report["early_watch_advanced"] == 1
+
+
+def test_recent_funnel_uses_actual_first_terminal_gate_for_choke(tmp_path):
+    funnel = tmp_path / "funnel-terminal.jsonl"
+    screening = tmp_path / "screening-terminal.jsonl"
+    summaries = tmp_path / "summaries-terminal.jsonl"
+    _write_jsonl(screening, [])
+    _write_jsonl(summaries, [])
+    rows = []
+    for index in range(3):
```

Ruff (0.16.3)

[warning] Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)